external/vcm/vcm/derived_mapping.py

```diff
@@ -89,7 +89,11 @@ def _recurse_find_deps(vars, deps):
 
 @DerivedMapping.register("cos_zenith_angle", required_inputs=["time", "lon", "lat"])
 def cos_zenith_angle(self):
-    return vcm.cos_zenith_angle(self["time"], self["lon"], self["lat"])
+    # TODO: eagerly loading the lon and lat inputs is required for the integration
+    # tests to pass.  It would be nice to remove this requirement now that
+    # cos_zenith_angle is dask compatible, but that would require figuring out the
+    # appropriate way to chunk the time coordinate.
+    return vcm.cos_zenith_angle(self["time"], self["lon"].load(), self["lat"].load())
```

Comment:
I'd normally complain about a load, but lon and lat should be small enough.

Reply:
Yeah, I agree, eventually it would be nice to remove this requirement, but it was happening silently before anyway, and at least this time the values will be computed much faster.